docs/command_line_tutorial.md

```diff
@@ -0,0 +1,90 @@
+
+Each output sub-directory will contain several JSON files that were generated during the corresponding run:
+
+- `per_instance_stats.json` contains a serialized list of `PerInstanceStats`, which contains the statistics produced for the metrics for each instance (i.e. input).
+- `run_spec.json` contains the `RunSpec`, which specifies the scenario, adapter and metrics for the run.
+- `scenario.json` contains a serialized `Scenario`, which contains the scenario for the run and specifies the instances (i.e. inputs) used.
+- `scenario_state.json` contains a serialized `ScenarioState`, which contains every request to and response from the model.
+- `stats.json` contains a serialized list of `PerInstanceStats`, which contains the statistics produced for the metrics, aggregated across all instances (i.e. inputs).
+
+The command also writes the file `benchmarking_output/runs/v1/run_specs.json`, which contains the run spec descriptions for all the runs. 
+
+`helm-run` provides additional arguments that can be used to filter out  `--models-to-run`, `--groups-to-run` and `--priority`. It can be convenient to create a large `run_specs.conf` file containing every run spec description of interest, and then use these flags to filter down the RunSpecs to actually run. As an example, the main `run_specs.conf` file used for the HELM benchmarking paper can be found [here](https://github.com/stanford-crfm/helm/blob/main/src/benchmark/presentation/run_specs.conf).
+
+## Using `helm-summarize`
+
+The `helm-summarize` reads the output files of `helm-run` and computes aggregate statistics. Run the following:
+
+```
+helm-summarize --suite v1
+```
+
+This reads the pre-existing files in `benchmark_output/runs/v1/` that were written by `helm-run` previously, and writes the following new files back to `benchmark_output/runs/v1/`:
+
+- `summary.json` contains a serialized `ExecutiveSummary` with a date and suite name.
+- `runs.json` contains serialized list of `Run`, which contains the run path, run spec and adapter spec and statistics for each run.
+- `groups.json` contains a serialized list of `Table`, each containing information about groups in a group category.
+- `groups_metadata.json` contains a list of all the groups along with a human-readable description and a taxonomy.
+
+Additionally, for each group and group-relavent metric, it will output a pair of files: `benchmark_output/runs/v1/groups/latex/<group_name>_<metric_name>.tex` and `benchmark_output/runs/v1/groups/latex/<group_name>_<metric_name>.json`. These files contain the statistics for that metric from each run within the group.
+
+**Note**: Because `helm-summarize` reads the `run_specs.json` that was produced by `helm-run` and `run_specs.json` is overwritten by every invocation of `helm-run`, `helm-summarize` will only process runs from the most recent invocation of `helm-run`.
```

Comment:
This might be confusing to people unless we say that you can run `helm-run` in parallel, but then we should pass in the flag to not generate the `run_specs.json`, but that might be too much...

Reply:
The other situation this arises if if you run using different run_specs in the same `benchmark_output` directory e.g. run `run_specs_small.conf` and then `run_specs_tiny.conf`, only the runs from the latest `helm-run` invocation shows up in the frontend, which can be confusing.